package-overrides/npm/redbox-react@1.1.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "registry": "npm",
```

Comment:
That's what I thought, but it wasn't installing the dependencies via npm without setting the registry to npm.